File: AccountantWeb/Models/Changes.cs
using System;

namespace Accountant.Models
{
    public class Changes
    {
        public int ID { get; set; }
        public DateTime Category { get; set; }
        public DateTime Expense { get; set; }
        public DateTime Report { get; set; }
        public DateTime ShoppingListItem { get; set; }
        public DateTime User { get; set; }
    }
}

File: AccountantAndroid/app/src/main/java/com/kunkliricsi/accountant/model/entities/Changes.java
package com.kunkliricsi.accountant.model.entities;

import java.util.Date;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "changes")
public class Changes {

    @PrimaryKey
    public int id;

    @ColumnInfo(name = "category")
    public Date Category;

    @ColumnInfo(name = "expense")
    public Date Expense;

    @ColumnInfo(name = "report")
    public Date Report;

    @ColumnInfo(name = "shoppinglistitem")
    public Date ShoppingListItem;

    @ColumnInfo(name = "user")
    public Date User;
}
